### PR TITLE
Staging to Master

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/useCases/getDownloadPolicyUrlInteractor.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const {
+  getDownloadPolicyUrlInteractor,
+} = require('./getDownloadPolicyUrlInteractor');
+
+describe('getDownloadPolicyUrlInteractor', () => {
+  beforeEach(() => {});
+
+  it('throw unauthorized error on invaliid role', async () => {
```
Comment (rachaelparris) [inline shared/src/business/useCases/getDownloadPolicyUrlInteractor.test.js]: Found a typo!

File: shared/src/business/useCases/getUploadPolicyUrlInteractor.test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+const { getUploadPolicyInteractor } = require('./getUploadPolicyInteractor');
+
+describe('getUploadPolicyInteractor', () => {
+  beforeEach(() => {});
+
+  it('throw unauthorized error on invaliid role', async () => {
```
Comment (rachaelparris) [inline shared/src/business/useCases/getUploadPolicyUrlInteractor.test.js]: Another typo

File: shared/src/business/useCases/processStreamRecordsInteractor.test.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  it('does not call index function if recordsToProcess is an empty array', async () => {
+    await processStreamRecordsInteractor({
+      applicationContext,
+      recordsToProcess: [],
+    });
+
+    expect(indexSpy).not.toHaveBeenCalled();
+  });
+
+  it('calls index function with correct params only for records with eventName "INSERT"', async () => {
```
Comment (rachaelparris) [inline shared/src/business/useCases/processStreamRecordsInteractor.test.js]: This test really needs to check both `INSERT` and `MODIFY`. I'll fix that.

File: shared/src/business/entities/docketEntry/DocketEntryFactory.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ DocketEntryFactory.VALIDATION_ERROR_MESSAGES = {
     'Enter a valid date received',
   ],
   eventCode: 'Select a document type',
-  lodged: 'Enter selection for Filing Status.',
+  lodged: 'Enter selection for Filing status.',
```
Comment (matthopson) [inline shared/src/business/entities/docketEntry/DocketEntryFactory.js]: Are we keeping `Filing` over `filing`?

File: shared/src/notifications/sendNotificationToUser.test.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    })
+    .mockImplementation(() => {
+      return { postToConnection };
+    }),
+  getPersistenceGateway: () => ({
+    getWebSocketConnectionsByUserId: getWebSocketConnectionsByUserIdMock,
+  }),
+};
+
+describe('update primary contact on a case', () => {
```
Comment (rachaelparris) [inline shared/src/notifications/sendNotificationToUser.test.js]: Description needs to be updated.

File: shared/src/business/test/createCaseInteractor.e2e.test.js [hunk 6]
```diff
@@ -72,7 +74,7 @@ describe('createCase integration test', () => {
       docketRecord: [
         {
           description: 'Petition',
-          filedBy: 'Rick Petitioner',
+          filedBy: 'Petr. Rick Petitioner',
```
Comment (matthopson) [inline shared/src/business/test/createCaseInteractor.e2e.test.js:77]: :pickle:

File: web-client/integration-tests/journey/petitionsClerkClearsSignatureFromDraftDocument.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+export default test => {
+  return it('Petitions clerk applies signature to a draft document', async () => {
```
Comment (rachaelparris) [inline web-client/integration-tests/journey/petitionsClerkClearsSignatureFromDraftDocument.js]: Update description?

File: web-client/src/presenter/actions/setConfirmEditModalStateAction.test.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import { runAction } from 'cerebral/test';
+import { setConfirmEditModalStateAction } from './setConfirmEditModalStateAction';
+
+describe('setConfirmEditModalStateAction', () => {
+  it('do', async () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/setConfirmEditModalStateAction.test.js]: Yes, it do.

File: shared/src/business/test/createCaseInteractor.e2e.test.js (hunk 6 above)
Comment (ericsorenson) [reply shared/src/business/test/createCaseInteractor.e2e.test.js:77]: I see what you did there.

File: web-client/src/presenter/computeds/caseDeadlineReportHelper.test.js [hunk 9]
```diff
@@ -106,7 +107,7 @@ describe('caseDeadlineReportHelper', () => {
 
   describe('sortByDateAndDocketNumber', () => {
     it('compares cases by docket number if dates are equal', () => {
-      let result = sortByDateAndDocketNumber(
+      let result = sortByDateAndDocketNumber(applicationContext)(
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/caseDeadlineReportHelper.test.js:110]: Sanity check - is this right?

Comment (ericsorenson) [reply web-client/src/presenter/computeds/caseDeadlineReportHelper.test.js:110]: Turns out, yes.

File: web-client/src/presenter/actions/setConfirmEditModalStateAction.test.js (hunk 8 above)
Comment (ericsorenson) [reply web-client/src/presenter/actions/setConfirmEditModalStateAction.test.js]: People don't think it be like it is.

File: web-client/integration-tests/journey/petitionsClerkClearsSignatureFromDraftDocument.js (hunk 7 above)
Comment (ericsorenson) [reply web-client/integration-tests/journey/petitionsClerkClearsSignatureFromDraftDocument.js]: Updated.

File: shared/src/notifications/sendNotificationToUser.test.js (hunk 5 above)
Comment (ericsorenson) [reply shared/src/notifications/sendNotificationToUser.test.js]: Updated.

File: shared/src/business/useCases/getUploadPolicyUrlInteractor.test.js (hunk 2 above)
Comment (ericsorenson) [reply shared/src/business/useCases/getUploadPolicyUrlInteractor.test.js]: Updated.

File: shared/src/business/useCases/getDownloadPolicyUrlInteractor.test.js (hunk 1 above)
Comment (ericsorenson) [reply shared/src/business/useCases/getDownloadPolicyUrlInteractor.test.js]: Updated.

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/entities/docketEntry/DocketEntryFactory.js (hunk 4 above)
Comment (alsmith1) [reply shared/src/business/entities/docketEntry/DocketEntryFactory.js]: filing please

Comment (ericsorenson) [reply shared/src/business/entities/docketEntry/DocketEntryFactory.js]: Good question!

Comment (klohman) [reply shared/src/business/entities/docketEntry/DocketEntryFactory.js]: it should be "filing"

Comment (ericsorenson) [reply shared/src/business/entities/docketEntry/DocketEntryFactory.js]: Updated.

**Pull request merged**